tests/cephadm/test_service_stop.py

```diff
@@ -0,0 +1,25 @@
+from cli.cephadm.cephadm import CephAdm
```

Comment:
Generalize this test case workflow. Change name to `test_cluster_service_operations.py`. Use config parameter in test case config to pass operation.